Buildings/Fluid/HeatExchangers/Validation/WetCoilEffectivenessNTU.mo

```diff
@@ -0,0 +1,249 @@
+  connect(X_w2.y[1], souAir.Xi_in[1]) annotation (Line(points={{169,-80},{160,-80},
+          {160,-84},{142,-84}}, color={0,0,127}));
+  connect(souWat2.ports[1], hexDis.port_a1)
+    annotation (Line(points={{-160,80},{-40,80},{-40,72},{-30,72}},
+                                                    color={0,127,255}));
+  connect(hexDis.port_b1, sinWat.ports[2]) annotation (Line(points={{-10,72},{
+          40,72},{40,20},{60,20},{60,18}},   color={0,127,255}));
+  connect(X_w2.y[1], souAir2.Xi_in[1]) annotation (Line(points={{169,-80},{160,-80},
+          {160,36},{142,36}}, color={0,0,127}));
+  connect(souAir2.ports[1], hexDis.port_a2) annotation (Line(points={{120,40},{
+          50,40},{50,60},{-10,60}},    color={0,127,255}));
+  connect(hexWetNTU.port_b2, TDryBulOut.port_a) annotation (Line(points={{-30,0},
+          {-50,0}},                     color={0,127,255}));
+  connect(hexWetNTU.port_a2, RelHumIn.port_b) annotation (Line(points={{-10,0},
+          {0,0},{0,-80},{20,-80}},color={0,127,255}));
+  connect(hexDis.port_b2, TDryBulOut1.port_a)
+    annotation (Line(points={{-30,60},{-50,60}}, color={0,127,255}));
+  connect(senMasFraOut1.port_b, sinAir.ports[2]) annotation (Line(points={{-130,
+          60},{-140,60},{-140,52},{-160,52},{-160,48}}, color={0,127,255}));
+  connect(TDryBulOut.port_b, RelHumOut_eps.port_a) annotation (Line(points={{
+          -70,0},{-76,0},{-76,0},{-80,0}}, color={0,127,255}));
+  connect(RelHumOut_eps.port_b, senMasFraOut.port_a)
+    annotation (Line(points={{-100,0},{-110,0}}, color={0,127,255}));
+  connect(TDryBulOut1.port_b, RelHumOut_dis.port_a)
+    annotation (Line(points={{-70,60},{-80,60}}, color={0,127,255}));
+  connect(RelHumOut_dis.port_b, senMasFraOut1.port_a)
+    annotation (Line(points={{-100,60},{-110,60}}, color={0,127,255}));
+  annotation (Diagram(coordinateSystem(preserveAspectRatio=true,
+    extent={{-200,-120},{200,120}})),
+    experiment(
+      StopTime=100,
+      Tolerance=1e-06),
+    __Dymola_Commands(
+    file="Resources/Scripts/Dymola/Fluid/HeatExchangers/Validation/WetCoilEffectivenessNTU.mos"
+  "Simulate and plot"),
+  Documentation(info="<html>
+<p>
+This model duplicates an example from Mitchell and Braun 2012, example SM-2-1
+(Mitchell and Braun 2012) to validate a single case for the
+<a href=\"modelica://Buildings.Fluid.HeatExchangers.WetCoilEffectivenessNTU\">
+Buildings.Fluid.HeatExchangers.WetCoilEffectivenessNTU</a>
+model.
+</p>
+<h4>Validation</h4>
+<p>
+The example is a steady-state analysis of a wet coil with constant air
+and water inlet temperature and mass flow rate, and an increasing air inlet
+humidity which triggers the transition from a fully-dry to a fully-wet regime.
+The reference used for validation is the published experimental data.
+A discretized wet coil model is also simulated for comparison.
+</p>
+<p>
+Note that the outlet air relative humidity may slightly exceed 100% when using
+the epsilon-NTU model. 
```

Comment:
I added that because if we don't find a fix, at least the user should be made aware of that.
May even be useful in the documentation of the coil model itself.